share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -105,6 +105,10 @@ installpkg kbd kbd-misc
 ## required for anaconda-dracut (img-lib etc.)
 installpkg tar xz curl bzip2
 
+## this is only recommended by librsvg2, but anaconda crashes without it
+## see https://bodhi.fedoraproject.org/updates/FEDORA-2023-f069ac712b
+installpkg rsvg-pixbuf-loader
```

Comment:
You could also just say:

```suggestion
## required to display SVG images
installpkg rsvg-pixbuf-loader
```

i.e. it would be needed even without the crash

Reply:
OK, I'll tweak that.

One thing I think we maybe can tell from the backtrace: it's trying to display an icon which *only* exists (in the installer environment at least) as an SVG. Frame #11 mentions input-keyboard-symbolic , and in the installer env, the only file with that in its name is `/usr/share/icons/Adwaita/symbolic/devices/input-keyboard-symbolic.svg`. IIRC there is some kinda hidden magic fallback to a non-symbolic version somewhere in the codepath, but even with that, the only `input-keyboard` icons we have are also SVGs, there is no PNG `input-keyboard*` anywhere.

So maybe there's a path where GTK just assumes it will definitely be able to render the icon somehow once it's been found, but because we don't have the SVG renderer and we don't have any non-SVG icons, that assumption breaks? Or something along those lines?

Reply:
comment and commit message updated, thanks.

Reply:
It's not possible to know without understanding the missing frame in the backtrace. What I can say for sure is that it shouldn't crash inside gtk_widget_show_all().

Reply:
yeah, that does seem odd. I tried following it out from that end but it's a long way. it gets into glib deciding whether the thing is a widget, which involves trying to instantiate it as one, best as I can tell...

I think the thing to do might be to try and write a simpler reproducer which doesn't involve python and gobject introspection, maybe? just...see what happens if you do stuff with an icon that's present, but only as SVG, without the SVG pixbuf loader being there...